src/style.css

```diff
@@ -1 +1,118 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
```

Comment:
The use of the universal `*` selector can impact performance. It's recommended to avoid using this selector.